.travis.yml

```diff
@@ -15,10 +15,26 @@ before_install:
     # * See https://cloud.google.com/container-registry/docs/pulling-cached-images
     echo '{ "registry-mirrors": ["https://mirror.gcr.io"] }' | sudo tee /etc/docker/daemon.json
     sudo service docker restart
+script: ./build_image && docker run --rm openzipkin/java:test -version
+
+# Test on both archs
```

Comment:
@odidev this is how I was envisioning the build matrix to work in zipkin's travis. Run both archs to test, but only deploy on amd64. Note: the push if condition will be different in zipkin, and probably needs thinking through.